docs/pages/admin-guides/deploy-a-cluster/license.mdx

```diff
@@ -115,6 +115,34 @@ The Teleport Auth Service checks the expiration status of your license every
 hour. This means that, after you replace your license file and restart your Auth
 Service instances, it can take up to an hour to confirm the new license.
 
+If you are running a self-hosted Teleport Enterprise cluster on Kubernetes using
+the `teleport-cluster` Helm chart, complete the following steps to update your
+license:
+
+1. Download a new license file from your Teleport account using the instructions
+   we provide [earlier in this guide](#download-your-license-file). We assume
+   that the license file is called `license.pem`.
+
+1. Delete the secret that stores your existing license. We assume that the
+   secret is called `license`, but there may be a different secret name
+   specified in the `licenseSecretName` field of the chart's values file.
```

Comment:
The instructions are missing the namespace. This is a very common mistake our customers are doing: 
 they deploy Teleport in the `teleport` namespace, update the license in the `default` namespace, the license expires and they open a support ticket.